forest/src/cli/wallet_cmd.rs

```diff
@@ -0,0 +1,137 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use super::stringify_rpc_err;
+use rpc_client::{balance, default, export, list, new, new_client, set_default, sign};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub enum WalletCommands {
+    /// Prints out wallet balance of specified address via RPC
+    #[structopt(about = "Get account balance")]
+    Balance {
+        #[structopt(short, help = "Input a valid address")]
+        address: String,
+    },
+
+    /// Creates wallet of specified signature type via RPC
+    #[structopt(
+        about = "Generate a new key of the given type",
+        help = "Generate a new key of the given type"
+    )]
+    New {
+        #[structopt(
+            short,
+            default_value = "secp256k1",
+            help = "Input signature type [bls | secp256k1] (default secp256k1)"
+        )]
+        sig_type: String,
+    },
+
+    /// Prints out all addresses in the wallet via RPC
+    #[structopt(about = "List wallet address", help = "List wallet address")]
+    List,
+
+    /// Marks the given address as as the default wallet via RPC
+    #[structopt(
+        about = "Set default wallet address",
+        help = "Set default wallet address"
+    )]
+    SetDefault {
+        #[structopt(short, help = "Input valid address")]
+        address: String,
```

Comment:
oh can you make this a positional argument as well? Doesn't make sense why you need the flag here?